cmd/clusterctl/config/assets/cert-manager.yaml

```diff
@@ -1,26 +1,42 @@
-# This is manually edited cert-manager v0.11.0 release.
```

Comment:
@wfernandes could you kindly double check we are not introducing regressions here?

Reply:
I took a look over the changes you'd made, and all of those changes are now part of the upstream release and should no longer be required (although if there were any more edits beyond what's documented in this comment, I cannot be sure).

Reply:
Yeah...and we got tests to prove it 😉 YAY!